xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/WikisLiveTableResultsMacros.xml

```diff
@@ -94,7 +94,7 @@
       #set($tableAlias = $order)
     #end
     ## only safe version should be used in SQL queries
-    #set($safe_tableAlias = "$!{tableAlias.replaceAll('[^a-zA-Z0-9_.]','')}")
+    #set($safe_tableAlias = $tableAlias)
     #livetable_getTableAlias($safe_tableAlias)
```

Comment:
I think

    #livetable_getTableAlias($tableAlias)

works too, no? You shouldn't have to set the variable before.